.travis.yml

```diff
@@ -14,6 +14,7 @@ before_install:
   # Override Xenial's default Java version (github.com/travis-ci/travis-ci/issues/10290)
   - export PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
   - export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+  - pip install --user protobuf
```

Comment:
I see two deprecation warnings during this step with suggestions for how to address them: https://travis-ci.org/ampproject/amphtml/jobs/538444937#L310

Reply:
I think updating to Python 3 would be more than trivial (and is beyond my knowledge of Python). I think it's something that should be done by EoY though, so I'd avoid investing too much into this Travis setup.

Regarding the pip warning: that's probably an issue with xenial as well - and we can't use `pip install --upgrade pip` to fix it since `pip` is installed by Apt so that command would require `sudo`.

Reply:
Agree with your assessment here.

Reply:
Opened #22555 as a tracking issue for Python 3.